src/System.CommandLine.Tests/Help/HelpBuilderTests.cs

```diff
@@ -842,6 +842,55 @@ public void Option_argument_descriptor_indicates_enums_values(Type type)
             _console.Out.ToString().Should().Contain($"--opt <Read|ReadWrite|Write>{_columnPadding}{description}");
         }
 
+        [Fact]
+        public void Command_argument_default_value_provided()
```

Comment:
I'd suggest renaming these tests to clarify the scenario and include the expectation, e.g. "Help describes default values for command arguments"

Reply:
So, "Command_argument_default_value_provided" will be renamed to "Help_desribes_default_value_for_single_command_argument"

Reply:
That looks perfect. 

The naming that we tend toward in this project describes the behavior in plain language, favors clarity over conciseness, and avoids focusing on type names and implementation details. This helps the tests function as a spec and prevents us from having to rename tests unless we're changing a behavior (as opposed to refactoring).